### PR TITLE
closes issue #4303 - update tests by removing method call to createSa…

### PR DESCRIPTION
**Pull request by sharath2106**
…ndbox as sinon is a sandbox by default

### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions.

### Description of the Change

The change made in the PR is to use `sinon@5.0.0` which is a sandbox by default. Previously, we were creating a sandbox and using it across tests. This change would remove the intermediate layer of creating sandbox as it is no longer needed.

### Alternate Designs

No alternatives

### Benefits

`Sinon` is a sandbox by default in `sinon@5.0.0`

### Possible Drawbacks

No side-effects.

### Applicable issues

Fixes the issue - https://github.com/mochajs/mocha/issues/4303

It's not a major or minor release.

### CODE REVIEW

**Comment by sharath2106** [comment]
@nicojs Is there anything to do with `createSandbox()` in `index.d.ts`? 

Screenshot 2020-06-04 at 10 18 34 PM

**Comment by sharath2106** [comment]
And I could see the `licence/cla` check pending even though I've signed the CLA when prompted. Am I missing something?

**Comment by juergba** [comment]
Have you tried to do this:

> Gopishankar Haridas seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please add the email address used for this commit to your account.

File: test/node-unit/buffered-runner.spec.js [hunk 1]
```diff
@@ -13,11 +13,10 @@ const BUFFERED_RUNNER_PATH = require.resolve(
 );
 const Suite = require('../../lib/suite');
 const Runner = require('../../lib/runner');
-const {createSandbox} = require('sinon');
+var sinon = require('sinon');
```
Comment (wnghdcjfe) [inline test/node-unit/buffered-runner.spec.js:16]: How about `const sinon = require('sinon')`

Comment (sharath2106) [reply test/node-unit/buffered-runner.spec.js:16]: Missed changing this. Will update the PR.

**Comment by juergba** [comment]
I don't know. First try this: `If you have already a GitHub account, please add the email address used for this commit to your account.`

If it doesn't work: reset your commit locally (`git reset <commit-nr>`), commit again your changes and then force-push to remote (`git push -f ...`). Your email adresses should match at that point of course.

**Comment by juergba** [comment]
@sharath2106 another rebase to `master` should solve the _netlify_ error, sorry.

**Comment by juergba** [comment]
@sharath2106 I changed the `author` of your first commit, rebased onto `master` and forced-pushed to your branch. The _netlify_ and _licence_ checks seem to be fine now.
Could you have another look onto this and confirm, please?

**Pull request merged**